estraverse/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for estraverse
+// Project: https://github.com/estools/estraverse
+// Definitions by: Sanex3339 <https://github.com/sanex3339>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'estraverse' {
```

Comment:
do not add an ambient module declaration, i.e.:

```ts
export interface Visitor {...}
```
instead of 
```ts
declare module 'estraverse' {
     export interface Visitor {...}
}
```